lib/src/main/java/com/shopify/checkoutkit/CheckoutDialog.kt

```diff
@@ -119,6 +119,7 @@ internal class CheckoutDialog(
         }
 
         show()
+        checkoutWebView.isPresented()
```

Comment:
nit: the difference is potentially just ms, so not a blocker, but I think calling within `setOnShowListener {  }` would be a bit more accurate than right after `show` 🤔